docs/src/main/tut/typeclasses/arrow.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: docs
+title:  "Arrow"
+section: "typeclasses"
+source: "core/src/main/scala/cats/arrow/Arrow.scala"
+scaladoc: "#cats.arrow.Arrow"
+---
+# Arrow
+
+`Arrow` is a type class for modeling something that behaves like functions. For example, `scala.Function1`(`A => B`), `cats.data.Kleisli`(wrapping an `A => F[B]`, also known as `ReaderT`), and `cats.data.Cokleisli`(wrapping an `F[A] => B`), have `Arrow` instances.
```

Comment:
Sorry I am late in the game.  This is kind of a nit but I am not so sure about this very first sentence. 

> `Arrow` is a type class for modeling something that behaves like functions.

I kind of feel that this is not the original intention of the type class. `Arrow` is, well, arrow in category theory. function is just one of the possible definitions.  So a more precise way to put it might be 
```markdown
`Arrow` is a type class for modeling composable relationship between two types. One example of such composable relationship is function `A => B`, other examples includes `cats.data.Kleisli`(wrapping an `A => F[B]`, also known as `ReaderT`), and `cats.data.Cokleisli`(wrapping an `F[A] => B`), have `Arrow` instances.
```
WDYT?

Reply:
sgtm. Thanks for the correction.